perl-package/AI-MXNetCAPI/mxnet.i

```diff
@@ -349,7 +349,7 @@ int MXGetGPUCount(int* out);
  * \param total_mem pointer to the integer holding total GPU memory
  * \return 0 when success, -1 when failure happens
  */
-int MXGetGPUMemoryInformation(int dev, int *out, int *out);
+int MXGetGPUMemoryInformation(int dev, uint64_t *out, uint64_t *out);
```

Comment:
@szha Given the usage I'd consider this a non-breaking change, but it is technically changing types to an argument.  From a SemVer perspective are you ok with changing this in a minor update?

Reply:
I also pondered this question and it's worth discussing.  As the MXGetGPUMemoryInformation interface stands now, it would only report global memory total and free bytes < 2MB correctly.  It's been many years since GPUs had so little global memory, so it's hard to imagine anyone out there using an interface so broken.  MXNet didn't wrap this C++ API in Python, although it was exposed in Perl I think.

Reply:
Yeah I also grepped through the code and only see references to this API being used from Perl.

Reply:
I think @DickJC123 you meant 2GB? One alternative is to provide the new interface as MXGetGPUMemoryInformation64 and update the old interface in major version